hadoop-ozone/dist/src/main/smoketest/omha/data-creation-before-om-bootstrap.robot

```diff
@@ -0,0 +1,97 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Smoke test for creating data needed for om bootstrap load test.
+Resource            ../commonlib.robot
+Test Timeout        5 minutes
+Test Setup          Run Keyword if    '${SECURITY_ENABLED}' == 'true'    Kinit test user     testuser     testuser.keytab
+
+*** Variables ***
+${VOLUME}           vol1
+${BUCKET}           bucket1
+${SNAP_1}           snap1
+${SNAP_2}           snap2
+${KEY_PREFIX}       sn
+${KEY_1}            key1
+${KEY_2}            key2
+${TMP_FILE}         tmp.txt
+
+*** Keywords ***
+Create volume and bucket
+    [arguments]         ${volume}           ${bucket}
+    ${vol_res} =        Execute             ozone sh volume create /${volume}
+                        Should Be Empty     ${vol_res}
+    ${bucket_res} =     Execute             ozone sh bucket create /${volume}/${bucket}
+                        Should Be Empty     ${bucket_res}
+
+Create tmp file
+    [arguments]         ${file_name}
+    ${create_res} =     Execute             touch ${file_name}
+                        Should Be Empty     ${create_res}
+    ${ls_grep_res} =    Execute             ls -lah | grep '${file_name}'
+                        Should contain      ${ls_grep_res}      ${file_name}
+
+Delete tmp file
+    [arguments]         ${file_name}
+    Execute             rm ${file_name}
+    ${file_exists} =    Execute             [[ -f ${file_name} ]] && echo "File exists" || echo "File doesn't exist"
+                        Should contain      ${file_exists}      File doesn't exist
+
+Create a key and set contents same as the keyName
+    [arguments]         ${volume}           ${bucket}      ${key_prefix}       ${key_name}         ${tmp_file}
+    Execute             > ${tmp_file}
```

Comment:
I dont understand the purpose of this tmp file.  why not just use the name of the key?